internal/logfields/fields.go

```diff
@@ -0,0 +1,41 @@
+package logfields
+
+// Identifiers
+const (
```

Comment:
nit: I think conventionally we would put these all in one big const block, with blank lines separating different categories.